manifests/node_exporter.pp

```diff
@@ -86,36 +86,37 @@
 #  The binary release version
 
 class prometheus::node_exporter (
-  String $download_extension,
```

Comment:
Those two parameters are mandatory. They are also defined in `data/defaults.yaml`.